pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
     'gwcs >=0.18.1',
     'jsonschema >=4.8',
     'numpy >=1.22',
-    'photutils @ git+https://github.com/astropy/photutils.git',
+    'photutils >=1.9.0',
```

Comment:
The tests are failing, @bmorris3 does this require us to wait for a photutils release. If so can one happen? If not then the `romancal.lib.tests.test_psf` tests will need to be xfailed.

Reply:
Larry is making the release this afternoon.